control-station/src/components/SensorBoxes/Console.tsx

```diff
@@ -1,14 +1,37 @@
+import { useContext, useEffect, useRef, useState } from "react";
 import "./SensorData.css";
+import PodContext from "@/services/PodContext";
 
 function Console() {
+	const { podData } = useContext(PodContext);
+	const [stateList, setStateList] = useState<string[]>([]);
+	const listEndRef = useRef<HTMLLIElement | null>(null);
+
+	useEffect(() => {
+		if (podData.state) {
+			setStateList((prev) => [...prev, podData.state]);
+		}
+	}, [podData.state]);
```

Comment:
Issue: while this may work for displaying messages just about the state, we'll have more messages in practice (e.g. about faults) which won't be compatible with the approach here. See the description on #56 for the suggested approach of adding a property to `PodData`.